<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobilityConstraintsDialog</class>
 <widget class="QDialog" name="MobilityConstraintsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Limit mobility constraints</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="qcb_useMobilityConstraints">
     <property name="text">
      <string>Use limit mobility constraints</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CopiableItemsTableView" name="qtvb_currentConstraints"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Estimated mobilities whose value exceeds the constraints of the previous ionic form will be displayed in red.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ql_info">
     <property name="text">
      <string>NOTE: Keep in mind that the constraints currently displayed are valid only for the initial estimates and will change during fit.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopiableItemsTableView</class>
   <extends>QTableView</extends>
   <header>src/ui/copiableitemstableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
